### PR TITLE
Short Article on Container Performance Study 

### PR DESCRIPTION
**Pull request by gqlo**
Short Article on Container Performance Study

### CODE REVIEW

**Comment by KoheiIchikawa** [comment]
Could you also add "Cloud" as a tag so that it would be shown as one of the articles of Cloud researches?

**Comment by gqlo** [comment]
> Could you also add "Cloud" as a tag so that it would be shown as one of the articles of Cloud researches?

Sure, I will do that.

File: content/ja/project/virtualization-performance/index.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+## Publication
+- Guoqing Li, Keichi Takahashi, Kouhei Ichikawa, Hajimu Iida, Pree Thiengburanathum, Passakorn Phannachitta, “Comparative Performance Study of Lightweight Hypervisors Used in Container Environment”, 11th International Conference on Cloud Computing and Services Science(CLOSER 2021), Apr. 2021.
```
Comment (keichi) [inline content/ja/project/virtualization-performance/index.md:57]: A space is needed before the open parenthesis `(`.

**Review by keichi — CHANGES_REQUESTED** [review]
Please remove `.index.md.swp`. This is a swap file generated by vim.

File: content/ja/project/virtualization-performance/index.md (hunk 1 above)
Comment (gqlo) [reply content/ja/project/virtualization-performance/index.md]: sure, I just resolved those two issues.

**Review by keichi — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**